test/NuGet.Configuration.Test/SettingsTests.cs

```diff
@@ -1791,6 +1791,31 @@ public void GetGlobalPackagesFolder_FromNuGetConfig()
         }
 
         [Fact]
+        public void GetGlobalPackagesFolder_FromNuGetConfig_RelativePath()
+        {
+            // Arrange
+            var config = @"<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+<config>
+<add key=""globalPackagesFolder"" value=""..\..\NuGetPackages"" />
+</config>
+</configuration>";
+
+            var nugetConfigPath = "NuGet.config";
+            using (var mockBaseDirectory = TestFilesystemUtility.CreateRandomTestFolder())
+            {
+                TestFilesystemUtility.CreateConfigurationFile(nugetConfigPath, mockBaseDirectory, config);
+                Settings settings = new Settings(mockBaseDirectory);
+
+                // Act
+                var globalPackagesFolderPath = SettingsUtility.GetGlobalPackagesFolder(settings);
```

Comment:
No. With https://github.com/NuGet/Home/issues/1062, if globalPackagesFolder is relative, full path is constructed relative to solution directory. exe throws if solution directory is not available (Either the switch or.sln file)